layouts/index.headers

```diff
@@ -0,0 +1,11 @@
+ /*
+  X-Frame-Options: DENY
+  X-XSS-Protection: 1; mode=block
+  X-Content-Type-Options: nosniff
+  Referrer-Policy: origin-when-cross-origin
+
+
+# HTTP/2 server push config, Netlify Link headers https://www.netlify.com/blog/2017/07/18/http/2-server-push-on-netlify/
+/dist/*
```

Comment:
You're requesting that H2 Push only happen when a page is requested from the `/dist/` path.  I don't think that's what you're going for.

If you move these under the `/*` section, will Netlify send `Link` headers with non-HTML content?  We don't want to push extra content if the user is requesting an image or XML file, for example.

We may just want to scope it to the homepage (`/`) for now.